### PR TITLE
Address a11y violations in Settings screens

### LINKED ISSUE #967: [A11y] Address A11y issues on Settings
This ticket will look to address issues of an accessibility (a11y) nature in the Settings section of the main nav. Note: for simplicity of this ticket we will not cover Custom Images.

> Get the [Chrome Dev Tools Axe Extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
> Running this extension on the page first loaded and as you use the page will help reveal what needs to be fixed

## Areas covered by this ticket

> Note you need to be a dashboard admin; being part of the ODH Dashboard Config `adminGroups` setting.

Main nav:

* Settings => Cluster Settings
* Settings => User Management

## Data setup for testing

There is no additional data setup needed for these pages.

### PR DESCRIPTION
**Pull request by jenny-s51**
## Description

Closes #967 

## How Has This Been Tested?

User and Group Settings - Before:
Screen Shot 2023-02-23 at 3 53 58 PM

User and Group Settings - After:
Screen Shot 2023-02-27 at 9 45 12 AM

Cluster Settings - no violations:
Screen Shot 2023-02-23 at 3 52 41 PM

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: frontend/src/components/MultiSelection.tsx [hunk 1]
```diff
@@ -11,9 +11,10 @@ import { MenuItemStatus } from 'pages/groupSettings/groupTypes';
 type MultiSelectionProps = {
   value: MenuItemStatus[];
   setValue: (itemSelection: MenuItemStatus[]) => void;
+  bodyText: string;
```
Comment (Gkrumbach07) [inline frontend/src/components/MultiSelection.tsx:14]: nit pick. `bodyText` can be confusing as MultiSelection does not render a body text at all. When reusing the component, someone might think "whatever i put in body text will be used a visible label". I would rename it to something like ariaLabel or allow users to pass `...props` into the `<Select {...props}>` component.

Comment (andrewballantyne) [reply frontend/src/components/MultiSelection.tsx:14]: I agree with Gage here -- I think the simplest solution would be to just make the prop `ariaLabel` instead, as anything prefixed with aria is going to be clear to be a11y support.

Comment (jenny-s51) [reply frontend/src/components/MultiSelection.tsx]: Thanks @Gkrumbach07 @andrewballantyne, updated

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
Looks like the Dropdown itself has an aria label issue. 
image

Seems the dropdown itself needs one -- not sure if PF allows us to configure that.

**Comment by lucferbux** [comment]
> Looks like the Dropdown itself has an aria label issue. image
> 
> Seems the dropdown itself needs one -- not sure if PF allows us to configure that.

Verified that the aria label is now included

Screenshot 2023-03-01 at 12 46 55

**Review by lucferbux — APPROVED** [review]
/lgtm

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**